### PR TITLE
docs(Upload): polish the info and examples

### PR DESCRIPTION
**Pull request by tujoworker**
I think, before announcement, the docs needs some more info about how we want it to be used.

Any thoughts and suggestion that can improve it further?

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/upload/info.md [hunk 1]
```diff
@@ -4,6 +4,39 @@ showTabs: true
...
 ## Description
 
-The Upload should be used in scenarios where the user has to upload any kind of files.
+The Upload component should be used in scenarios where the user has to upload any kind of files.
+
+## How to use the Upload component
+
+- Files selected by the user should be uploaded immediately (temporary location).
+- The user should be able to remove them (files) during the session.
+- If the Upload component is shown in a submit form, then a [GlobalStatus](/uilib/components/global-status) should be a part of the form.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/upload/info.md:13]: Maybe we could/should provide an example? 🤔

**Review by langz — APPROVED** [review]
Great 👍 
I can try to go over all the documentation for the upload component, and see if I find any possible improvements tomorrow 🙏

File: packages/dnb-design-system-portal/src/docs/uilib/components/upload/info.md (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/upload/info.md:13]: Yes! That would be nice 👍 we have to check that out, and find a system where we can make a demo.

**Pull request merged**